lib/python-parser/metadata-parser.ts

```diff
@@ -0,0 +1,47 @@
+import { specifierValidRange } from "./common";
+import { PythonPackage, PythonRequirement } from "./types";
+
+const PACKAGE_NAME = "Name: ";
+const PACKAGE_VERSION = "Version: ";
+const PACKAGE_DEPS = "Requires-Dist: ";
+const DEP_PARSE_REGEX = /^(?<name>[\w.-]+)(\s?\(?(?<specifier><|<=|!=|==|>=|>|~=|===)(?<version>[\w.]+)\)?)?/;
+export function getPackageInfo(fileContent: string): PythonPackage {
+  const lines = fileContent.split("\n");
+  let name = "";
+  let version = "";
+  const dependencies: PythonRequirement[] = [];
+  for (let line of lines) {
+    line = line.trim();
+    if (line.length === 0) {
+      continue;
+    }
+    if (line.startsWith(PACKAGE_NAME)) {
+      name = line.substring(PACKAGE_NAME.length);
```

Comment:
Do you need it with the `Name` etc prefix or am I missing something? :D

Reply:
Not sure I understand the question. If the line starts with `Name: `, then the name of the package is what comes after that

Reply:
Ignore, misread it :D